src/popup/components/Protocol/index.tsx

```diff
@@ -23,6 +25,10 @@ function ProtocolState() {
     getProtocolOptIn().hasCompletedLiveness(),
   );
 
+  const mnemonicSaved = useLoadedState(() =>
+    getMnemonicSave().challengeNeeded(),
+  );
+
```

Comment:
Name variable to match the terminology: `isChallengeNeeded`. Also methods that return booleans should usually start with `is`, `has`, `does`, or something similar. For this case, `isChallengeNeeded` would be good.